src/opm/output/eclipse/Summary.cpp

```diff
@@ -88,6 +88,28 @@
 
 #include <fmt/format.h>
 
+template <> struct fmt::formatter<Opm::EclIO::SummaryNode::Category>: fmt::formatter<string_view> {
+    // parse is inherited from formatter<string_view>.
+    template <typename FormatContext>
+    auto format(Opm::EclIO::SummaryNode::Category c, FormatContext& ctx) {
```

Comment:
I thought there already was a way to convert a category to a string, but apparently not.  Thanks for making this!

Reply:
Please be aware that this is only available in this *.cc file for now.

Reply:
> Please be aware that this is only available in this *.cc file for now.

Right.

> We always used `static_cast<int>` to make it compile with newer fmt/compiler versions.

Yes, I just thought we'd already made a function of the form
```C++
std::string Category2String(const EclIO::SummaryNode::Category cat)
```
but there is no such function in our codebase.